app.js

```diff
@@ -0,0 +1,56 @@
+//refs: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Node_server_without_framework
+var http = require('http');
+var fs = require('fs');
+var path = require('path');
+
+http.createServer(function (request, response) {
+    console.log('request ', request.url);
+
+    var filePath = '.' + request.url;
+    if (filePath == './') {
+        filePath = './index.html';
+    }
+
+    var extname = String(path.extname(filePath)).toLowerCase();
+    var mimeTypes = {
+        '.html': 'text/html',
+        '.js': 'text/javascript',
+        '.css': 'text/css',
```

Comment:
Please validate if all these are being used, Lets keep which are being used currently

Reply:
now added a boilerplate for the app. will update as per the user specification.